cassovary-core/src/main/scala/com/twitter/cassovary/graph/MemoryMappedDirectedGraph.scala

```diff
@@ -51,48 +70,63 @@ class MemoryMappedDirectedGraph(file: File) extends DirectedGraph[Node] {
   private class MemoryMappedDirectedNode(override val id: Int) extends Node {
     val nodeOutboundOffset = outboundOffset(id)
     val nodeInboundOffset = inboundOffset(id)
-    override def outboundNodes(): Seq[Int] = new IndexedSeq[Int] {
```

Comment:
Pankaj didn't like the overrides (I guess Cassovary has the convention of not using override unless you're actually overriding and implementation.)  I didn't feel strongly, so I removed the override.